Model/Page.php

```diff
@@ -134,6 +134,11 @@
     protected $target;
 
     /**
+     * @var int
+     */
+    protected $targetRedirectType = 302;
```

Comment:
Could you please explain your reasoning as all the other fields are `protected` too?